### PR TITLE
Switch Puppeter to Playwright

### PR DESCRIPTION
**Pull request by shirshak55**
Changes puppeteer repo to playwright as puppeteer is almost abandoned.

### CODE REVIEW

**Comment by shirshak55** [comment]
After merging this PR this repo will be stable  and will follow semver.

File: src/fastPage/pageStealth.ts [hunk 1]
```diff
@@ -68,8 +70,8 @@ const getChromeRuntimeMock = (window: any) => {
   }
 }
 
-async function runtimeStealth(page: any) {
-  await page.evaluateOnNewDocument(
+async function runtimeStealth(page: Page) {
+  await page.addInitScript(
```
Comment (shirshak55) [inline src/fastPage/pageStealth.ts]: need to do more work on stealth. I will do more once i get my laptop repaired.

**Comment by shirshak55** [comment]
i have made a special version 0.0.1 for testing this. After v2 it will follow semver and should be very safe to use.

**Comment by shirshak55** [comment]
Waiting playwright to be abit stable.

**Comment by janisblaus** [comment]
fyi, following your work here :) helped me a lot!

**Comment by shirshak55** [comment]
@janisblaus  

Thanks :)

I am currently waiting for proxy features, screencast, print feature etc. from upstream playwright repo. Once that is released i  will merge it and make this repo stable.

**Comment by shirshak55** [comment]
i have uploaded special version 0.0.2 for initial testing which contains fixes for firefox

**Comment by janisblaus** [comment]
Ive noted that iframeStealth() gets detected by some services, e.g. - speedtest.net displays a blank page if this is enabled. I guess this is not the right place to post this, but wasn't sure if this is an issue at all

**Comment by shirshak55** [comment]
@janisblaus try without stealth. After that check console and can u show us error ? and these site uses chrome to detect so may be u should try firefox it is working well for me.

**Pull request merged**